test/server.cpp

```diff
@@ -162,52 +161,53 @@ std::ostream& operator<<(std::ostream& out, const Resource& r)
+  { WITH_ETAG, "/ROOT/skin/jquery-ui/external/jquery/jquery.js" },
+  { WITH_ETAG, "/ROOT/skin/jquery-ui/jquery-ui.theme.min.css" },
+  { WITH_ETAG, "/ROOT/skin/jquery-ui/jquery-ui.min.css" },
+  { WITH_ETAG, "/ROOT/skin/taskbar.js" },
+  { WITH_ETAG, "/ROOT/skin/taskbar.css" },
+  { WITH_ETAG, "/ROOT/skin/block_external.js" },
 
-  { NO_ETAG,   "/catalog/root.xml" },
-  { NO_ETAG,   "/catalog/searchdescription.xml" },
-  { NO_ETAG,   "/catalog/search" },
+  { NO_ETAG,   "/ROOT/catalog/root.xml" },
+  { NO_ETAG,   "/ROOT/catalog/searchdescription.xml" },
+  { NO_ETAG,   "/ROOT/catalog/search" },
 
-  { NO_ETAG,   "/search?content=zimfile&pattern=a" },
+  { NO_ETAG,   "/ROOT/search?content=zimfile&pattern=a" },
 
-  { NO_ETAG,   "/suggest?content=zimfile" },
-  { NO_ETAG,   "/suggest?content=zimfile&term=ray" },
+  { NO_ETAG,   "/ROOT/suggest?content=zimfile" },
+  { NO_ETAG,   "/ROOT/suggest?content=zimfile&term=ray" },
 
-  { NO_ETAG,   "/catch/external?source=www.example.com" },
+  { NO_ETAG,   "/ROOT/catch/external?source=www.example.com" },
 
-  { WITH_ETAG, "/zimfile/A/index" },
-  { WITH_ETAG, "/zimfile/A/Ray_Charles" },
+  { WITH_ETAG, "/ROOT/zimfile/A/index" },
+  { WITH_ETAG, "/ROOT/zimfile/A/Ray_Charles" },
 };
 
 const ResourceCollection resources200Uncompressible{
-  { WITH_ETAG, "/skin/jquery-ui/images/animated-overlay.gif" },
-  { WITH_ETAG, "/skin/caret.png" },
-
-  { WITH_ETAG, "/meta?content=zimfile&name=title" },
-  { WITH_ETAG, "/meta?content=zimfile&name=description" },
-  { WITH_ETAG, "/meta?content=zimfile&name=language" },
-  { WITH_ETAG, "/meta?content=zimfile&name=name" },
-  { WITH_ETAG, "/meta?content=zimfile&name=tags" },
-  { WITH_ETAG, "/meta?content=zimfile&name=date" },
-  { WITH_ETAG, "/meta?content=zimfile&name=creator" },
-  { WITH_ETAG, "/meta?content=zimfile&name=publisher" },
-  { WITH_ETAG, "/meta?content=zimfile&name=favicon" },
-  { WITH_ETAG, "/meta?content=zimfile&name=Illustration_48x48@1" },
-
-  { WITH_ETAG, "/zimfile/I/m/Ray_Charles_classic_piano_pose.jpg" },
-
-  { WITH_ETAG, "/corner_cases/A/empty.html" },
-  { WITH_ETAG, "/corner_cases/-/empty.css" },
-  { WITH_ETAG, "/corner_cases/-/empty.js" },
+  { WITH_ETAG, "/ROOT/skin/jquery-ui/images/animated-overlay.gif" },
+  { WITH_ETAG, "/ROOT/skin/caret.png" },
+
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=title" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=description" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=language" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=name" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=tags" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=date" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=creator" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=publisher" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=favicon" },
+  { WITH_ETAG, "/ROOT/meta?content=zimfile&name=Illustration_48x48@1" },
+  { NO_ETAG, "/ROOT/catalog/v2/illustration/zimfile?size=48" },
```

Comment:
This should probably be `WITH_ETAG`
But https://github.com/kiwix/libkiwix/blob/master/src/server/internalServer.cpp#L320-L328 explicitly prevent catalog endpoints to be cacheable.
@veloman-yunkan want do you think, should we made (some) catalog endpoints cacheable ? Maybe in another PR ?

Reply:
@mgautierfr @veloman-yunkan This is the topic of https://github.com/kiwix/libkiwix/issues/650 which expects your both feedbacks.

Reply:
> @veloman-yunkan want do you think, should we made (some) catalog endpoints cacheable ? Maybe in another PR ?

> @mgautierfr @veloman-yunkan This is the topic of #650 which expects your both feedbacks.

Agree. Let's leave it for #650.